src/Notes.cc

```diff
@@ -354,6 +354,15 @@ NotePairs::appendNewOnly(const NotePairs *src)
     }
 }
 
+void
+NotePairs::replaceOrAddOrAppend(const NotePairs *src, const std::vector<SBuf> &appendKeys)
+{
+    for (auto e: src->entries)
+        if (std::find(appendKeys.begin(), appendKeys.end(), e->name()) == appendKeys.end())
+            remove(e->name());
+    append(src);
```

Comment:
```suggestion
    }
    append(src);
```

Reply:
I will add braces here but please note that there are other similar functions which dont use braces for  "for" loop. (toString, findFirst, hasPair etc.) (although I will not make changes there unless you tell me to do so)

Reply:
Done

Reply:
Thank you. Multiline loop bodies in new and changed code should use braces in most cases for clarity and safety sake. Old code should not be changed just for the sake of adding the missing braces.

Most rules have some exceptions, especially when it comes to such vague things as clarity and safety. It is difficult and probably both impractical and unnecessary to codify everything exactly, but there are many "easy" cases like this one where it is fairly clear what the right thing to do is. Thank you for doing the right thing.